testng-core/src/test/java/test/priority/parallel/EfficientPriorityParallelizationTest2.java

```diff
@@ -100,9 +100,9 @@ public void setUp() {
     suiteOneTestMethodLevelEventLogs = getTestMethodLevelEventLogsForSuite(SUITE_A);
 
     suiteOneTestOneTestMethodLevelEventLogs =
-        getTestMethodLevelEventLogsForTest(SUITE_A, SUITE_A_TEST_A);
-    suiteOneTestOneTestMethodLevelEventLogs.stream()
-        .filter(e -> e.getEvent() == TestNgRunEvent.LISTENER_TEST_METHOD_START);
```

Comment:
It turns out these lines were doing nothing, however, it is not clear what was the intention.
The code has been added in https://github.com/testng-team/testng/pull/1910/files#diff-af16c31109bb20a887788e673417872d9ce0789634234868c6bcfaa1c13203b6R92, and it was not modified since.

@MicahLC, do you by chance remember what was the intention of adding event filtering of "event == listener_test_method_start" here? It looks like result of the stream was not later, so the code looks like a dead code.

Reply:
@vlsi Exactly. I have to investigate what are the potential side effects. The alternative will be easy: just remove the dead code :)

Reply:
Yeah. Removing dead code is one of the approaches. However, it might signal the test is not doing something expected.

What do you think of removing the dead code, and creating an issue to investigate if it should be re-added?

Reply:
Yes. I will consider it when it will be the last failing test :)

Reply:
How dare you make me revisit my code from 5 years ago 😄 
This was probably an oversight on my part, I think the stream/filter here isn't necessary. On line 268 of this file (as written), it checks `e.getEvent() == TestNgRunEvent.LISTENER_TEST_METHOD_PASS`, and there won't be any events of that type if you filter by `e.getEvent() == TestNgRunEvent.LISTENER_TEST_METHOD_START` here.

If I had to guess, I probably had some logging there originally and accidentally left this line in.